src/java/org/apache/cassandra/service/accord/AccordKeyspace.java

```diff
@@ -0,0 +1,843 @@
+                builder.addCell(live(CommandsColumns.writes, timestampMicros, serialize(command.writes.get(), CommandSerializers.writes, version)));
+            }
+
+            if (command.result.hasModifications())
+            {
+                builder.addCell(live(CommandsColumns.result_version, timestampMicros, versionBytes));
+                builder.addCell(live(CommandsColumns.result, timestampMicros, serialize((AccordData) command.result.get(), AccordData.serializer, version)));
+            }
+
+            if (command.waitingOnCommit.hasModifications())
+            {
+                addStoredMapChanges(builder, CommandsColumns.waiting_on_commit,
+                                    timestampMicros, nowInSeconds, command.waitingOnCommit,
+                                    AccordKeyspace::serializeTimestamp, bytes -> bytes);
+            }
+
+            if (command.blockingCommitOn.hasModifications())
+            {
+                addStoredSetChanges(builder, CommandsColumns.blocking_commit_on,
+                                    timestampMicros, nowInSeconds, command.blockingApplyOn,
+                                    AccordKeyspace::serializeTimestamp);
+            }
+
+            if (command.waitingOnApply.hasModifications())
+            {
+                addStoredMapChanges(builder, CommandsColumns.waiting_on_apply,
+                                    timestampMicros, nowInSeconds, command.waitingOnApply,
+                                    AccordKeyspace::serializeTimestamp, bytes -> bytes);
+            }
+
+            if (command.blockingApplyOn.hasModifications())
+            {
+                addStoredSetChanges(builder, CommandsColumns.blocking_apply_on,
+                                    timestampMicros, nowInSeconds, command.blockingApplyOn,
+                                    AccordKeyspace::serializeTimestamp);
+            }
+
+            if (command.storedListeners.hasModifications())
+            {
+                addStoredSetChanges(builder, CommandsColumns.listeners,
+                                    timestampMicros, nowInSeconds, command.storedListeners,
+                                    ListenerProxy::identifier);
+            }
+            ByteBuffer key = CommandsColumns.keyComparator.make(command.commandStore().generation(),
+                                                                command.commandStore().index(),
+                                                                serializeTimestamp(command.txnId())).serializeAsPartitionKey();
+            PartitionUpdate update = PartitionUpdate.singleRowUpdate(Commands, key, builder.build());
+            return new Mutation(update);
+        }
+        catch (IOException e)
+        {
+            throw new RuntimeException(e);
+        }
+    }
+
+    private static ByteBuffer serializeKey(PartitionKey key)
+    {
+        UUIDSerializer.instance.serialize(key.tableId().asUUID());
```

Comment:
this line doesn't use the value... and redoes it in the next line... can we remove?